### ISSUE TITLE
ng g @co-it/schematics:prettier

### ISSUE THREAD

**Issue by GregOnNet** [co-IT/schematics#1]
- [x] Install prettier
- [x] Provide opinionated .prettierrc
- [x] Register `format` command in package.json
- [x] Update `tslint.json` to extend from [tslint-config-prettier](https://www.npmjs.com/package/tslint-config-prettier).
- [x] Warn if competing prettier configuration is detected
- [x] Update tslint rules according to prettier
- [x] Offer possibility to setup pre commit hook uing husky
- [x] Place husky configuration in own file (see [Angular Material Husky](https://github.com/angular/material2/blob/master/.huskyrc.json))
- [x] Provide workaround to override an existing `.prettierrc`
  - Using `apply` with `MergeStrategie.Overwrite` does not work (see [#11337](https://github.com/angular/angular-cli/issues/11337#issuecomment-473350357)).

**Comment by GregOnNet** [comment]
**tslint.json**
```json
{
 "quotemark": [true, "single"],
 "arrow-parens": false
}
```

**Comment by GregOnNet** [comment]
#11 is merged.
